docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1156 @@
+       See ``Z2Symmetries`` :ref:`example <z2_sym>` for updated workflow.
+
+   * - :class:`~qiskit.opflow.primitive_ops.Z2Symmetries`
+     - :class:`~qiskit.quantum_info.Z2Symmetries`. See example :ref:`below <z2_sym>`.
+
+.. _pauli_sum_op:
+
+Example 1: ``PauliSumOp``
+##############################
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Opflow</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.opflow import PauliSumOp
+    from qiskit.quantum_info import SparsePauliOp, Pauli
+
+    qubit_op = PauliSumOp(SparsePauliOp(Pauli("XYZY"), coeffs=[2]), coeff=-3j)
+
+.. raw:: html
+
+   </details>
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Alternative</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.quantum_info import SparsePauliOp, Pauli
+
+    qubit_op = SparsePauliOp(Pauli("XYZY")), coeff=-6j)
+
+.. raw:: html
+
+   </details>
+
+.. _z2_sym:
+
+Example 2: ``Z2Symmetries`` and ``TaperedPauliSumOp``
+#####################################################
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Opflow</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.opflow import PuliSumOp, Z2Symmetries, TaperedPauliSumOp
+
+    qubit_op = PauliSumOp.from_list(
+        [
+        ("II", -1.0537076071291125),
+        ("IZ", 0.393983679438514),
+        ("ZI", -0.39398367943851387),
+        ("ZZ", -0.01123658523318205),
+        ("XX", 0.1812888082114961),
+        ]
+    )
+    z2_symmetries = Z2Symmetries.find_Z2_symmetries(qubit_op)
+    tapered_op = z2_symmetries.taper(qubit_op)
+    # can be represented as:
+    tapered_op = TaperedPauliSumOp(primitive, z2_symmetries)
+
+.. raw:: html
+
+   </details>
+
+.. raw:: html
+
+    <details>
+    <summary><a><b>Alternative</b></a></summary>
+
+.. code-block:: python
+
+    from qiskit.quantum_info import SparsePauliOp, Z2Symmetries
+
+    qubit_op = SparsePauliOp.from_list(
+        [
+            ("II", -1.0537076071291125),
+            ("IZ", 0.393983679438514),
+            ("ZI", -0.39398367943851387),
+            ("ZZ", -0.01123658523318205),
+            ("XX", 0.1812888082114961),
+        ]
+    )
+    z2_symmetries = Z2Symmetries.find_z2_symmetries(qubit_op)
+    tapered_op = z2_symmetries.taper(qubit_op)
+
+.. raw:: html
+
+   </details>
+
+ListOps
+~~~~~~~
+*Back to* `Index`_
+
+The :mod:`~qiskit.opflow.list_ops` module contained classes for manipulating lists of :mod:`~qiskit.opflow.primitive_ops`
+or :mod:`~qiskit.opflow.state_fns`. The :mod:`~qiskit.quantum_info` alternatives for this functionality are the
+:class:`~qiskit.quantum_info.PauliList`, :class:`~qiskit.quantum_info.SparsePauliOp` (for sums of ``Pauli``\s).
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+
+   * - :class:`~qiskit.opflow.list_ops.ListOp`
+     - No direct replacement. This is the base class for operator lists. In general, these could be replaced with
```

Comment:
a lot of these have no direct replacement, is it worth adding a sentence to the paragraph above about why no direct replacement is needed? Or why the choice was made not to replace them?